src/onNextTick.js

```diff
@@ -0,0 +1,36 @@
+let timeout;
+const timeoutQueue = [];
+
+export default function onNextTick(cb) {
```

Comment:
That's true, but I think it is not actually accurate because of the way the JS event loop functions.